src/index.js

```diff
@@ -140,41 +129,45 @@ module.exports = {
     'no-useless-concat': 'error',
     'no-var': 'error',
     'no-void': 'error',
-    'no-warning-comments': 'off',
-    'no-with': 'off',
-    'object-curly-spacing': 'off',
+    'no-warning-comments': 'error',
+    'no-with': 'error',
+    'object-curly-spacing': ['error', 'always'],
     'object-shorthand': 'error',
-    'one-var': 'off',
+    'one-var': ['error', 'never'],
     'operator-assignment': 'error',
-    'operator-linebreak': 'off',
-    'padded-blocks': 'off',
+    'operator-linebreak': 'error',
```

Comment:
The default behaviour for this rule is to allow linebreak `after`. Do we really want this?
I would set this to `operator-linebreak: ["error", "none"]` instead?

WDYT @ruiquelhas ?